src/huntsman/pocs/camera/pyro/client.py

```diff
@@ -225,7 +227,9 @@ def autofocus(self, blocking=False, *args, **kwargs):
                 a fine focus. Default False.
             make_plots (bool, optional: Whether to write focus plots to images folder, default
                 False.
-            blocking (bool, optional): Whether to block until autofocus complete, default False.
+            blocking (bool, optional): Whether to block (on the client-side) until autofocus
+                complete, default False.
+            timeout (float, optional): The client-side autofocus timeout. Default 5 min.
```

Comment:
We should come up with a standard way of documenting these "if `None` then check config else use fallback of 600 seconds" kind of statements.  The default here can change without any code changes, which is a bit misleading.  I'm guilty of this in many other places.

```suggestion
            timeout (float, optional): The client-side autofocus timeout. The default value of `None` will lookup the `focusing.<focus_type>.timeout` value in the config server. If not provided, a default fallback of 600 seconds is used.
```